x-pack/solutions/search/plugins/serverless_search/public/navigation_tree.ts

```diff
@@ -167,10 +167,44 @@ export const navigationTree = ({ isAppRegistered }: ApplicationStart): Navigatio
             }),
           },
           {
-            link: 'management',
+            id: 'stack_management',
```

Comment:
This creates the user experience of has an icon of an arrow pointing to the right, which opens a side panel with Stack Management sub-navigation.